telemetry/opentelemetry-config/src/test/java/io/helidon/telemetry/otelconfig/TestServiceBootstrap.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.telemetry.otelconfig;
+
+import io.helidon.common.media.type.MediaTypes;
+import io.helidon.config.Config;
+import io.helidon.config.ConfigSources;
+import io.helidon.service.registry.GlobalServiceRegistry;
+import io.helidon.service.registry.Lookup;
+import io.helidon.service.registry.Service;
+import io.helidon.service.registry.Services;
+
+import io.opentelemetry.api.OpenTelemetry;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.notNullValue;
+
+class TestServiceBootstrap {
+
+    @Test
+    void testServiceBootstrap() {
+
+        Config config = Config.just(ConfigSources.create(
+                """
+                        telemetry:
+                          service: "test-otel"
+                          global: false
+                          signals:
+                            tracing:
+                              sampler:
+                                type: "always_on"
+                              exporters:
+                                - type: otlp
+                                  protocol: http/proto
+                                  name: my-oltp
+                                - type: zipkin
+                              processors:
+                                - max-queue-size: 21
+                                  type: batch
+                        """,
+                MediaTypes.APPLICATION_YAML));
+
+        Services.set(Config.class, config);
```

Comment:
If you want to use `Services.set` in testing, you should annotate your class as `@Testing.Test`, as otherwise you will not be able to call it more than once (the annotation ensures that you run within a different global service instance).